 2019-08-20 13:05:55,047 [P17396/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:05:55,069 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:05:55,087 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 17ms)
 2019-08-20 13:05:55,152 [P17396/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:05:55,153 [P17396/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:05:55,156 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:05:55,479 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 323ms)
 2019-08-20 13:05:55,479 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:05:55,485 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2019-08-20 13:05:55,501 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:05:55,502 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-08-20 13:05:55,503 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:05:55,512 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 9ms)
 2019-08-20 13:05:55,513 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:05:55,513 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:05:55,514 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:05:55,523 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 9ms)
 2019-08-20 13:05:55,541 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:05:55,541 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:05:55,541 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:05:55,548 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 7ms)
 2019-08-20 13:05:55,549 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:05:55,549 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:05:55,549 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:05:55,556 [P17396/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 6ms)
 2019-08-20 13:05:57,209 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   bei System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:05:57,211 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:05:57,402 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:05:57,402 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:05:58,413 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:05:58,413 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:05:59,413 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:05:59,413 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:06:00,415 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:06:00,415 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:06:01,416 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:fff458cd-085c-4a1c-81fc-92c5283a6a19
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:06:01,416 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:06:02,419 [P17396/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   bei System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:772bff73-3184-4380-ae5b-901ada09bde2
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:06:02,420 [P17396/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:06:02,423 [P17396/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   bei Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   bei Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   bei Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   bei Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   bei Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-08-20 13:06:12,628 [P17396/D2/T9] INFO  Umbraco.Core.MainDom - Stopping...
 2019-08-20 13:06:12,628 [P17396/D2/T9] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-08-20 13:06:12,630 [P17396/D2/T9] INFO  Umbraco.Web.ExamineStartup - Complete (took 1ms)
 2019-08-20 13:06:12,631 [P17396/D2/T9] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-08-20 13:14:52,319 [P20884/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:14:52,347 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:14:52,353 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 5ms)
 2019-08-20 13:14:52,432 [P20884/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:14:52,437 [P20884/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:14:52,438 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:14:52,795 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 356ms)
 2019-08-20 13:14:52,795 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:14:52,822 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 26ms)
 2019-08-20 13:14:52,838 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:14:52,838 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:14:52,838 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:14:52,844 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 5ms)
 2019-08-20 13:14:52,844 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:14:52,845 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:14:52,845 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:14:52,851 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2019-08-20 13:14:52,870 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:14:52,870 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:14:52,870 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:14:52,875 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 4ms)
 2019-08-20 13:14:52,876 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:14:52,876 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:14:52,876 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:14:52,882 [P20884/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 5ms)
 2019-08-20 13:14:54,731 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   bei System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:54,733 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:54,885 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:54,885 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:55,886 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:55,886 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:56,890 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:56,890 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:57,891 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:57,891 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:58,892 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Es konnte eine Verbindung mit dem Server hergestellt werden, doch während des Anmeldevorgangs trat ein Fehler auf. (provider: Shared Memory Provider, error: 0 - Kein Prozess ist am anderen Ende der Pipe.) ---> System.ComponentModel.Win32Exception (0x80004005): Kein Prozess ist am anderen Ende der Pipe
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2b6a769a-5b16-4e5d-8d1d-77c3f6ef720c
Fehlernummer (Error Number):233,Status (State):0,Klasse (Class):20
 2019-08-20 13:14:58,892 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:59,901 [P20884/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost\SQLEXPRESS, InitialCatalog: Umbraco_DB. Exception: System.Data.SqlClient.SqlException (0x80131904): Fehler bei der Anmeldung für den Benutzer "umbAddi".
   bei System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   bei System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   bei System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   bei System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   bei System.Data.SqlClient.SqlConnection.Open()
   bei Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:059d8677-c645-4e20-9b8a-25bd5b9bee84
Fehlernummer (Error Number):18456,Status (State):1,Klasse (Class):14
 2019-08-20 13:14:59,901 [P20884/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-08-20 13:14:59,904 [P20884/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   bei Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   bei Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   bei Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   bei Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   bei Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-08-20 13:15:10,000 [P20884/D2/T12] INFO  Umbraco.Core.MainDom - Stopping...
 2019-08-20 13:15:10,001 [P20884/D2/T12] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-08-20 13:15:10,002 [P20884/D2/T12] INFO  Umbraco.Web.ExamineStartup - Complete (took 1ms)
 2019-08-20 13:15:10,002 [P20884/D2/T12] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-08-20 13:17:07,165 [P21276/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:17:07,182 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:17:07,191 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 8ms)
 2019-08-20 13:17:07,263 [P21276/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:17:07,264 [P21276/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:17:07,265 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:07,595 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 329ms)
 2019-08-20 13:17:07,595 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:17:07,601 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2019-08-20 13:17:07,614 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:07,614 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:07,614 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:17:07,619 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 4ms)
 2019-08-20 13:17:07,620 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:07,620 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:07,620 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:17:07,630 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 9ms)
 2019-08-20 13:17:07,652 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:07,652 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:07,653 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:17:07,662 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 9ms)
 2019-08-20 13:17:07,663 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:07,663 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:07,663 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:17:07,668 [P21276/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 5ms)
 2019-08-20 13:17:09,348 [P21276/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-08-20 13:17:09,794 [P21276/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-08-20 13:17:09,886 [P21276/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-08-20 13:17:09,973 [P21276/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-08-20 13:17:09,988 [P21276/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 2903ms)
 2019-08-20 13:17:10,345 [P21276/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:61026/umbraco
 2019-08-20 13:17:10,346 [P21276/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:61026/umbraco (UmbracoModule request)
 2019-08-20 13:17:10,792 [P21276/D2/T7] INFO  umbraco.content - Load Xml from file...
 2019-08-20 13:17:10,804 [P21276/D2/T7] INFO  umbraco.content - Loaded Xml from file.
 2019-08-20 13:17:11,861 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:11,861 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:11,861 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 13:17:11,867 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 5ms)
 2019-08-20 13:17:15,888 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:15,889 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:15,889 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-08-20 13:17:15,903 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 15ms)
 2019-08-20 13:17:34,808 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:34,809 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:34,810 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:17:34,817 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 7ms)
 2019-08-20 13:17:35,123 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:35,123 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:35,123 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-08-20 13:17:35,134 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 10ms)
 2019-08-20 13:17:48,909 [P21276/D2/T7] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username philipp.bonny@googlemail.com from IP address ::1
 2019-08-20 13:17:48,910 [P21276/D2/T7] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: philipp.bonny@googlemail.com logged in from IP address ::1
 2019-08-20 13:17:49,406 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:49,407 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:49,407 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-08-20 13:17:49,413 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 6ms)
 2019-08-20 13:17:49,422 [P21276/D2/T7] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 13:17:49,427 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:49,427 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:49,427 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:17:49,427 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:17:49,430 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:49,430 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:49,430 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-08-20 13:17:49,436 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 6ms)
 2019-08-20 13:17:49,456 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:49,457 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:49,457 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-08-20 13:17:49,463 [P21276/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 6ms)
 2019-08-20 13:17:49,941 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:17:49,942 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:17:49,942 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-08-20 13:17:49,948 [P21276/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 5ms)
 2019-08-20 13:18:11,148 [P21276/D2/T16] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-08-19 13:18:10
 2019-08-20 13:19:12,245 [P21276/D2/T19] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Slide1' with Id '1120' has been unpublished.
 2019-08-20 13:19:12,935 [P21276/D2/T19] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Slide 2' with Id '1121' has been unpublished.
 2019-08-20 13:19:13,476 [P21276/D2/T19] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Slider Normal' with Id '1118' has been unpublished.
 2019-08-20 13:19:13,887 [P21276/D2/T19] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Slider Ordner' with Id '1117' has been unpublished.
 2019-08-20 13:19:14,831 [P21276/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:19:14,831 [P21276/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:19:14,831 [P21276/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-08-20 13:19:14,839 [P21276/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 8ms)
 2019-08-20 13:19:15,460 [P21276/D2/T19] INFO  umbraco.content - Loading content from database...
 2019-08-20 13:19:19,721 [P21276/D2/T7] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:19:19,732 [P21276/D2/T7] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:19:20,947 [P21276/D2/T20] INFO  umbraco.content - Loading content from database...
 2019-08-20 13:19:25,058 [P21276/D2/T7] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:19:25,074 [P21276/D2/T7] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:19:26,465 [P21276/D2/T7] INFO  umbraco.content - Loading content from database...
 2019-08-20 13:19:30,591 [P21276/D2/T19] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:19:30,615 [P21276/D2/T19] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:19:57,166 [P21276/D2/T22] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:19:57,166 [P21276/D2/T22] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:19:57,166 [P21276/D2/T22] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-08-20 13:19:57,191 [P21276/D2/T22] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 24ms)
 2019-08-20 13:20:05,711 [P21276/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:20:05,711 [P21276/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:20:05,711 [P21276/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 13:20:05,711 [P21276/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2019-08-20 13:22:31,519 [P21276/D2/T37] ERROR Umbraco.Web.Editors.ContentController - Unhandled controller exception occurred
System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   at Umbraco.Web.Editors.ContentController.CheckPermissions(IDictionary`2 storage, IUser user, IUserService userService, IContentService contentService, IEntityService entityService, Int32 nodeId, Char[] permissionsToCheck, IContent contentItem)
   at Umbraco.Web.WebApi.Filters.EnsureUserPermissionForContentAttribute.OnActionExecuting(HttpActionContext actionContext)
   at System.Web.Http.Filters.ActionFilterAttribute.OnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
 2019-08-20 13:22:59,779 [P21276/D2/T8] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'dfsdfd' with Id '1173' has been unpublished.
 2019-08-20 13:23:05,495 [P21276/D2/T33] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:23:05,507 [P21276/D2/T33] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:23:24,346 [P18616/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:23:24,365 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:23:24,383 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 18ms)
 2019-08-20 13:23:24,456 [P18616/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:23:24,457 [P18616/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:23:24,460 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:24,882 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 422ms)
 2019-08-20 13:23:24,882 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:23:24,883 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 0ms)
 2019-08-20 13:23:24,926 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:24,927 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:24,927 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:23:24,927 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2019-08-20 13:23:24,928 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:24,928 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:24,928 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:23:24,929 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2019-08-20 13:23:24,956 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:24,958 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-08-20 13:23:24,958 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:23:24,959 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2019-08-20 13:23:24,959 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:24,959 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:24,959 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:23:24,960 [P18616/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2019-08-20 13:23:28,485 [P18616/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-08-20 13:23:28,813 [P18616/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-08-20 13:23:28,896 [P18616/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-08-20 13:23:28,983 [P18616/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-08-20 13:23:28,996 [P18616/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 4777ms)
 2019-08-20 13:23:28,998 [P18616/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:23:28,999 [P18616/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:23:28,999 [P18616/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:23:29,279 [P18616/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:61026/umbraco
 2019-08-20 13:23:29,280 [P18616/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:61026/umbraco (UmbracoModule request)
 2019-08-20 13:23:29,903 [P18616/D2/T11] INFO  umbraco.content - Load Xml from file...
 2019-08-20 13:23:29,928 [P18616/D2/T11] INFO  umbraco.content - Loaded Xml from file.
 2019-08-20 13:23:30,824 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:30,824 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:30,824 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 13:23:30,825 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2019-08-20 13:23:32,335 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:32,336 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:32,336 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-08-20 13:23:32,336 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 0ms)
 2019-08-20 13:23:39,638 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:39,639 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:39,639 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:23:39,639 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:23:39,849 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:39,850 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:39,850 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-08-20 13:23:39,851 [P18616/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 0ms)
 2019-08-20 13:23:41,801 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:41,802 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:41,802 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-08-20 13:23:41,802 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 0ms)
 2019-08-20 13:23:41,833 [P18616/D2/T13] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 13:23:41,848 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:41,848 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:41,848 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:23:41,848 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:23:41,851 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:41,851 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:41,851 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-08-20 13:23:41,851 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 0ms)
 2019-08-20 13:23:41,881 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:41,882 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:41,882 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-08-20 13:23:41,882 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 0ms)
 2019-08-20 13:23:42,957 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:23:42,958 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:23:42,958 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-08-20 13:23:42,964 [P18616/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 6ms)
 2019-08-20 13:24:23,395 [P18616/D2/T19] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact' with Id '1082' has been unpublished.
 2019-08-20 13:24:24,095 [P18616/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:24:24,095 [P18616/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:24:24,095 [P18616/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-08-20 13:24:24,095 [P18616/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 0ms)
 2019-08-20 13:24:28,346 [P18616/D2/T17] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:24:28,354 [P18616/D2/T17] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:24:29,342 [P18616/D2/T20] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Blog' with Id '1081' has been unpublished.
 2019-08-20 13:24:30,182 [P18616/D2/T19] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-08-19 13:24:30
 2019-08-20 13:24:34,078 [P18616/D2/T20] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:24:34,087 [P18616/D2/T20] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:24:34,531 [P18616/D2/T17] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Portfolio' with Id '1080' has been unpublished.
 2019-08-20 13:24:39,246 [P18616/D2/T20] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:24:39,256 [P18616/D2/T20] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:24:39,968 [P18616/D2/T12] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Services' with Id '1079' has been unpublished.
 2019-08-20 13:24:44,762 [P18616/D2/T11] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:24:44,770 [P18616/D2/T11] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:24:45,023 [P18616/D2/T12] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Feature' with Id '1078' has been unpublished.
 2019-08-20 13:24:49,777 [P18616/D2/T12] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:24:49,784 [P18616/D2/T12] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:25:19,852 [P18616/D2/T12] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Home' with Id '1056' has been published.
 2019-08-20 13:25:24,529 [P18616/D2/T26] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:25:24,537 [P18616/D2/T26] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:25:55,366 [P18616/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:25:55,366 [P18616/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:25:55,366 [P18616/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-08-20 13:25:55,367 [P18616/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 0ms)
 2019-08-20 13:29:41,119 [P18616/D2/T11] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Info' with Id '1184' has been published.
 2019-08-20 13:29:45,731 [P18616/D2/T12] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:29:45,744 [P18616/D2/T12] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:29:47,982 [P18616/D2/T31] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'About' with Id '1076' has been unpublished.
 2019-08-20 13:29:53,126 [P18616/D2/T16] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:29:53,138 [P18616/D2/T16] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:33:58,514 [P18616/D2/T20] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 13:38:10,193 [P8236/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:38:10,224 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:38:10,238 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 14ms)
 2019-08-20 13:38:10,310 [P8236/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:38:10,311 [P8236/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:38:10,313 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:10,611 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 298ms)
 2019-08-20 13:38:10,611 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:38:10,617 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2019-08-20 13:38:10,631 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:10,632 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:10,632 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:38:10,645 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 12ms)
 2019-08-20 13:38:10,645 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:10,646 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:10,646 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:38:10,651 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2019-08-20 13:38:10,663 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:10,663 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:10,663 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:38:10,667 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 4ms)
 2019-08-20 13:38:10,668 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:10,668 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:10,669 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:38:10,673 [P8236/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 4ms)
 2019-08-20 13:38:12,218 [P8236/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-08-20 13:38:12,564 [P8236/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-08-20 13:38:12,632 [P8236/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-08-20 13:38:12,797 [P8236/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-08-20 13:38:12,806 [P8236/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 2707ms)
 2019-08-20 13:38:12,809 [P8236/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:38:12,810 [P8236/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:38:12,811 [P8236/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:38:13,083 [P8236/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:61026/umbraco
 2019-08-20 13:38:13,084 [P8236/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:61026/umbraco (UmbracoModule request)
 2019-08-20 13:38:13,456 [P8236/D2/T7] INFO  umbraco.content - Load Xml from file...
 2019-08-20 13:38:13,464 [P8236/D2/T7] INFO  umbraco.content - Loaded Xml from file.
 2019-08-20 13:38:14,317 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:14,317 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:14,317 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 13:38:14,323 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 6ms)
 2019-08-20 13:38:48,914 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:48,914 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:48,914 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:38:48,932 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 17ms)
 2019-08-20 13:38:49,210 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:49,211 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:49,211 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-08-20 13:38:49,235 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 23ms)
 2019-08-20 13:38:51,514 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:51,515 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:51,515 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-08-20 13:38:51,551 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 35ms)
 2019-08-20 13:38:51,575 [P8236/D2/T11] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 13:38:51,579 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:51,581 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-08-20 13:38:51,581 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:38:51,581 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:38:51,584 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:51,584 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:51,584 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-08-20 13:38:51,596 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 11ms)
 2019-08-20 13:38:51,632 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:51,633 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:51,633 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-08-20 13:38:51,646 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 13ms)
 2019-08-20 13:38:53,026 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:53,027 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:38:53,027 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-08-20 13:38:53,042 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 15ms)
 2019-08-20 13:38:59,494 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:38:59,496 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-08-20 13:38:59,496 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-08-20 13:38:59,512 [P8236/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 15ms)
 2019-08-20 13:39:07,489 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:39:07,489 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:39:07,489 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-08-20 13:39:07,515 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 26ms)
 2019-08-20 13:39:13,557 [P8236/D2/T13] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-08-19 13:39:13
 2019-08-20 13:39:15,790 [P8236/D2/T7] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Info' with Id '1184' has been unpublished.
 2019-08-20 13:39:16,493 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:39:16,493 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:39:16,493 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-08-20 13:39:16,502 [P8236/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 8ms)
 2019-08-20 13:39:20,668 [P8236/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Suchergebnisse' with Id '1183' has been unpublished.
 2019-08-20 13:39:20,961 [P8236/D2/T7] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:39:20,995 [P8236/D2/T7] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:39:25,347 [P8236/D2/T25] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:39:25,353 [P8236/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:39:27,793 [P8236/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Home' with Id '1056' has been unpublished.
 2019-08-20 13:39:32,521 [P8236/D2/T25] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:39:32,528 [P8236/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:40:41,856 [P8236/D2/T22] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Home' with Id '0' has been published.
 2019-08-20 13:40:46,527 [P8236/D2/T31] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:40:46,534 [P8236/D2/T31] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:41:23,901 [P8236/D2/T32] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Suchergebnisse' with Id '0' has been published.
 2019-08-20 13:41:28,502 [P8236/D2/T27] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:41:28,512 [P8236/D2/T27] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:42:05,696 [P8236/D2/T22] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Info' with Id '0' has been published.
 2019-08-20 13:42:10,371 [P8236/D2/T42] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:42:10,394 [P8236/D2/T42] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:42:31,314 [P8236/D2/T32] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Suchergebnisse' with Id '1186' has been published.
 2019-08-20 13:42:35,227 [P8236/D2/T22] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Info' with Id '1187' has been published.
 2019-08-20 13:42:39,761 [P8236/D2/T29] INFO  umbraco.content - Save Xml to file...
 2019-08-20 13:42:39,769 [P8236/D2/T29] INFO  umbraco.content - Saved Xml to file.
 2019-08-20 13:45:41,347 [P8236/D2/T32] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:45:41,348 [P8236/D2/T32] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:45:41,348 [P8236/D2/T32] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Editors.IEditorValidator
 2019-08-20 13:45:41,364 [P8236/D2/T32] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Editors.IEditorValidator (took 15ms)
 2019-08-20 13:51:57,388 [P17600/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 13:51:57,412 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 13:51:57,446 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 34ms)
 2019-08-20 13:51:57,570 [P17600/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 13:51:57,571 [P17600/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 13:51:57,573 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:51:58,197 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 623ms)
 2019-08-20 13:51:58,197 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 13:51:58,198 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 0ms)
 2019-08-20 13:51:58,256 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:51:58,257 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:51:58,257 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 13:51:58,257 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2019-08-20 13:51:58,257 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:51:58,258 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:51:58,258 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 13:51:58,260 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2019-08-20 13:51:58,277 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:51:58,278 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:51:58,278 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 13:51:58,278 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2019-08-20 13:51:58,279 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:51:58,279 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:51:58,279 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 13:51:58,280 [P17600/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2019-08-20 13:52:02,929 [P17600/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-08-20 13:52:03,332 [P17600/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-08-20 13:52:03,422 [P17600/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-08-20 13:52:03,517 [P17600/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-08-20 13:52:03,531 [P17600/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 6311ms)
 2019-08-20 13:52:03,534 [P17600/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:52:03,535 [P17600/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:52:03,536 [P17600/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 13:52:03,836 [P17600/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:61026/umbraco
 2019-08-20 13:52:03,838 [P17600/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:61026/umbraco (UmbracoModule request)
 2019-08-20 13:52:04,551 [P17600/D2/T11] INFO  umbraco.content - Load Xml from file...
 2019-08-20 13:52:04,553 [P17600/D2/T11] INFO  umbraco.content - Loaded Xml from file.
 2019-08-20 13:52:05,580 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:05,580 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:05,581 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 13:52:05,581 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2019-08-20 13:52:21,767 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:21,767 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:21,767 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:52:21,767 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:52:21,913 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:21,913 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:21,914 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-08-20 13:52:21,914 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 0ms)
 2019-08-20 13:52:22,660 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:22,660 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:22,660 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-08-20 13:52:22,661 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 0ms)
 2019-08-20 13:52:22,712 [P17600/D2/T11] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 13:52:22,716 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:22,716 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:22,717 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-08-20 13:52:22,717 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-08-20 13:52:22,719 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:22,719 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:22,720 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-08-20 13:52:22,720 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 0ms)
 2019-08-20 13:52:22,752 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:22,752 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:22,752 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-08-20 13:52:22,753 [P17600/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 0ms)
 2019-08-20 13:52:23,824 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:23,825 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:23,825 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-08-20 13:52:23,835 [P17600/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 10ms)
 2019-08-20 13:52:28,150 [P17600/D2/T14] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:52:28,150 [P17600/D2/T14] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:52:28,151 [P17600/D2/T14] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-08-20 13:52:28,151 [P17600/D2/T14] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 0ms)
 2019-08-20 13:53:04,701 [P17600/D2/T14] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-08-19 13:53:04
 2019-08-20 13:55:50,209 [P17600/D2/T28] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 13:55:50,210 [P17600/D2/T28] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 13:55:50,210 [P17600/D2/T28] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-08-20 13:55:50,210 [P17600/D2/T28] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 0ms)
 2019-08-20 14:03:16,940 [P17600/D2/T37] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 14:13:42,855 [P17600/D2/T28] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 14:24:39,923 [P17600/D2/T75] WARN  Umbraco.Core.Services.ApplicationTreeService - The tree definition: <add initialize="true" sortOrder="0" alias="artikelTree" application="shopSection" title="Artikel" iconClosed="icon-folder" iconOpen="icon-folder-open" type="NetWebkonzepteUmbreco.Klassen.ArtikelTreeController, NetWebkonzepteUmbreco" /> could not be resolved to a .Net object type
 2019-08-20 14:36:27,831 [P21432/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-HAK12FI
 2019-08-20 14:36:27,853 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-08-20 14:36:27,868 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 15ms)
 2019-08-20 14:36:28,139 [P21432/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-08-20 14:36:28,141 [P21432/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-08-20 14:36:28,142 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:28,570 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 428ms)
 2019-08-20 14:36:28,571 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-08-20 14:36:28,577 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 6ms)
 2019-08-20 14:36:28,598 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:28,599 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:36:28,599 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-08-20 14:36:28,614 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 15ms)
 2019-08-20 14:36:28,614 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:28,615 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:36:28,615 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-08-20 14:36:28,624 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 9ms)
 2019-08-20 14:36:28,657 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:28,658 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:36:28,658 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-08-20 14:36:28,668 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 10ms)
 2019-08-20 14:36:28,669 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:28,670 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:36:28,670 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-08-20 14:36:28,676 [P21432/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 6ms)
 2019-08-20 14:36:30,591 [P21432/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-08-20 14:36:30,985 [P21432/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-08-20 14:36:31,070 [P21432/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-08-20 14:36:31,174 [P21432/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-08-20 14:36:31,184 [P21432/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 3464ms)
 2019-08-20 14:36:31,187 [P21432/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-08-20 14:36:31,187 [P21432/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 14:36:31,188 [P21432/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-08-20 14:36:31,565 [P21432/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:61026/umbraco
 2019-08-20 14:36:31,575 [P21432/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:61026/umbraco (UmbracoModule request)
 2019-08-20 14:36:32,075 [P21432/D2/T10] INFO  umbraco.content - Load Xml from file...
 2019-08-20 14:36:32,079 [P21432/D2/T10] INFO  umbraco.content - Loaded Xml from file.
 2019-08-20 14:36:33,148 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:36:33,148 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:36:33,148 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-08-20 14:36:33,180 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 32ms)
 2019-08-20 14:37:32,206 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-08-20 14:37:32,207 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-08-20 14:37:32,207 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-08-20 14:37:32,211 [P21432/D2/T12] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-08-19 14:37:32
 2019-08-20 14:37:32,240 [P21432/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 33ms)
